website/docs/d/elasticsearch_domain.html.markdown

```diff
@@ -0,0 +1,111 @@
+* `node_to_node_encryption` - Node-to-node encryption options. See below.
+* `cluster_config` - Cluster configuration of the domain, see below.
+* `snapshot_options` - Snapshot related options, see below.
+* `vpc_options` - VPC related options, see below. Adding or removing this configuration forces a new resource ([documentation](https://docs.aws.amazon.com/elasticsearch-service/latest/developerguide/es-vpc.html#es-vpc-limitations)).
+* `log_publishing_options` - Options for publishing slow logs to CloudWatch Logs.
+* `elasticsearch_version` - The version of Elasticsearch to deploy. Defaults to `1.5`
+* `tags` - A mapping of tags to assign to the resource
+
+**ebs_options** the following attributes are exported:
+
+* `ebs_enabled` - Whether EBS volumes are attached to data nodes in the domain
+* `volume_type` - The type of EBS volumes attached to data nodes.
+* `volume_size` - The size of EBS volumes attached to data nodes (in GB).
+**Required** if `ebs_enabled` is set to `true`.
+* `iops` - The baseline input/output (I/O) performance of EBS volumes
+	attached to data nodes. Applicable only for the Provisioned IOPS EBS volume type.
+
+**encrypt_at_rest** the following attributes are exported:
+
+* `enabled` - Whether to enable encryption at rest. If the `encrypt_at_rest` block is not provided then this defaults to `false`.
+* `kms_key_id` - The KMS key id to encrypt the Elasticsearch domain with. If not specified then it defaults to using the `aws/es` service KMS key.
+
+**cluster_config** the following attributes are exported:
+
+* `instance_type` - Instance type of data nodes in the cluster.
+* `instance_count` - Number of instances in the cluster.
+* `dedicated_master_enabled` - Indicates whether dedicated master nodes are enabled for the cluster.
+* `dedicated_master_type` - Instance type of the dedicated master nodes in the cluster.
+* `dedicated_master_count` - Number of dedicated master nodes in the cluster
+* `zone_awareness_enabled` - Indicates whether zone awareness is enabled.
+
+**node_to_node_encryption** the following attributes are exported:
+
+* `enabled` - Whether to enable node-to-node encryption. If the `node_to_node_encryption` block is not provided then this defaults to `false`.
+
+**vpc_options** the following attributes are exported:
+
+AWS documentation: [VPC Support for Amazon Elasticsearch Service Domains](https://docs.aws.amazon.com/elasticsearch-service/latest/developerguide/es-vpc.html)
+
+**Note** you must have created the service linked role for the Elasticsearch service to use the `vpc_options`.
+If you need to create the service linked role at the same time as the Elasticsearch domain then you must use `depends_on` to make sure that the role is created before the Elasticsearch domain.
+See the [VPC based ES domain example](#vpc-based-es) above.
```

Comment:
This looks to be copied from the resource and doesn't seem to make sense in the data context.